pom.xml

```diff
@@ -130,122 +133,135 @@
     <developer>
       <id>openzipkin</id>
       <name>OpenZipkin Gitter</name>
-      <url>https://gitter.im/openzipkin/zipkin</url>
+      <url>https://gitter.im/apache/incubator-zipkin</url>
     </developer>
   </developers>
 
+  <mailingLists>
+    <mailingList>
+      <name>Zipkin Developer List</name>
+      <post>dev@zipkin.apache.org</post>
+      <subscribe>dev-subscribe@zipkin.apache.org</subscribe>
+      <unsubscribe>dev-unsubscribe@zipkin.apache.org</unsubscribe>
+    </mailingList>
+    <mailingList>
+      <name>Zipkin Commits</name>
+      <post>commits@zipkin.apache.org</post>
+      <subscribe>commits-subscribe@zipkin.apache.org</subscribe>
+      <unsubscribe>commits-unsubscribe@zipkin.apache.org</unsubscribe>
+    </mailingList>
+  </mailingLists>
+
   <distributionManagement>
     <repository>
-      <id>bintray</id>
-      <url>https://api.bintray.com/maven/openzipkin/maven/zipkin/;publish=1</url>
+      <id>apache.releases.https</id>
+      <url>https://repository.apache.org/service/local/staging/deploy/maven2</url>
     </repository>
     <snapshotRepository>
-      <id>jfrog-snapshots</id>
-      <url>https://oss.jfrog.org/artifactory/oss-snapshot-local</url>
+      <id>apache.snapshots.https</id>
+      <url>https://repository.apache.org/content/repositories/snapshots/</url>
     </snapshotRepository>
   </distributionManagement>
 
   <issueManagement>
     <system>Github</system>
-    <url>https://github.com/openzipkin/zipkin/issues</url>
+    <url>https://github.com/apache/incubator-zipkin/issues</url>
   </issueManagement>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>io.zipkin.zipkin2</groupId>
+        <groupId>org.apache.zipkin.zipkin2</groupId>
```

Comment:
Maybe now is the time to drop the `zipkin2`, since my interpretation was that gave us the ability to have both on the classpath, this gives us that as well

Reply:
drop for `org.apache.zipkin.zipkin` ? remember `org.apache.zipkin.brave` etc will exist. The main thing is not mingling other projects in the same group ID as that can be troublesome